ui/src/components/query/builder/edit/dialogs/SaveCustomSetDialog.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <Button label="Save custom set" text severity="info" @click="showSaveCustomSetDialog = true" />
+  <Dialog v-model:visible="showSaveCustomSetDialog" modal header="Save custom set" :style="{ maxWidth: '50vw', backgroundColor: 'var(--surface-section)' }">
+    <form @submit="onSubmit" class="flex flex-column gap-2">
+      <div class="flex flex-column">
+        Scheme
+        <div class="flex">
+          <div class="flex-1">
+            <div class="flex flex-column">
+              <Dropdown
+                id="scheme"
+                v-bind="scheme"
+                type="text"
+                :class="{ 'p-invalid': errors.scheme }"
+                aria-describedby="text-error"
+                :options="schemeOptions"
+                option-label="name"
+                option-value="@id"
+                placeholder="Scheme"
+                class="flex-1"
+              />
+              <small class="p-error flex-1" id="text-error">{{ errors.scheme || "&nbsp;" }}</small>
+            </div>
+          </div>
+
+          <div class="flex-1">
+            <div class="flex flex-column">
```

Comment:
double nested class divs should be combined into 1 with correct flex functionality